policy/cnspec_policy.proto

```diff
@@ -343,7 +343,11 @@ message FrameworkGroup {
   string title = 24;
   PolicyGroupDocs docs = 25;
   repeated cnquery.explorer.Author authors = 26;
-  repeated cnquery.explorer.Author approvers = 27;
+  repeated cnquery.explorer.Author reviewers = 27;
+  // only applies to GroupType IGNORED/DISABLED
+  // Only if true ignore the group
+  // Which means that unapproved groups get handled as if they were approved
+  bool rejected = 28;
```

Comment:
do we have a use case for approved exception too? if thats true, it might better to define this as an enum state instead of a bool so we can also express that

Reply:
We currently only care about `rejected`, but still this is a nice idea. That will make parts of the code easier to read and more clear. Thanks.

Reply:
I changed the type. Could you please have another look?